spec/code_objects/constants_spec.rb

```diff
@@ -50,19 +50,30 @@
 
   describe :BUILTIN_EXCEPTIONS do
     it "includes all base exceptions" do
+      bad_names = []
       YARD::CodeObjects::BUILTIN_EXCEPTIONS.each do |name|
-        expect(eval(name)).to be <= Exception
+         next if name == 'ClosedQueueError' && RUBY_VERSION < '2.3'
+         bad_names << name unless eval(name) <= Exception
       end
+      expect(bad_names).to be_empty
     end
   end
 
   describe :BUILTIN_CLASSES do
     it "includes all base classes" do
+      if RUBY_VERSION < '2.3'
```

Comment:
Ditto for manual checks. It shouldn't matter if the list includes more than the base classes (for that version).